### PR TITLE
WIP: remove copy of QtPositionQuick.framework for 5.11

### PR DESCRIPTION
**Pull request by janiversen**
The manual copy of QtPositionQuick.framework in CMakeLists.txt gives 2 "cp" errors

macdeployqt does copy this framework.

Remove manual copy of QtPositionQuick.framework in CMakeLists.txt

Signed-off-by: Jan Iversen <jan@casacondor.com>

### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [x ] Build system change
- [ ] Documentation change
- [ ] Language translation

### CODE REVIEW

**Comment by dirkhh** [comment]
That build failure with no log is odd. Restarting.

File: CMakeLists.txt [hunk 1]
```diff
@@ -436,10 +436,6 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	install(CODE "execute_process(COMMAND cp -a ${_qt5Core_install_prefix}/qml/QtQuick.2 ${CMAKE_BINARY_DIR}/${APP_BUNDLE_DIR}/Contents/Resources/qml)")
 	install(CODE "execute_process(COMMAND cp -a ${_qt5Core_install_prefix}/qml/QtLocation ${CMAKE_BINARY_DIR}/${APP_BUNDLE_DIR}/Contents/Resources/qml)")
 	install(CODE "execute_process(COMMAND cp -a ${_qt5Core_install_prefix}/qml/QtPositioning ${CMAKE_BINARY_DIR}/${APP_BUNDLE_DIR}/Contents/Resources/qml)")
-	if(NOT Qt5Core_VERSION VERSION_LESS 5.11.0)
-		# and with Qt 5.11 we need another library that isn't copied by macdeployqt
-		install(CODE "execute_process(COMMAND cp -a ${_qt5Core_install_prefix}/lib/QtPositioningQuick.framework ${CMAKE_BINARY_DIR}/${APP_BUNDLE_DIR}/Contents/Frameworks)")
-	endif()
```
Comment (dirkhh) [inline CMakeLists.txt:442]: so this was fixed at some point - do we know when? Unless of course I was confused when I added this last year...

**Comment by dirkhh** [comment]
> That build failure with no log is odd. Restarting.

ok, that's resolved. Still wondering if you happen to know when this got fixed - because I believe that with your change Qt 5.11 would break (not that I care TOO much... reasonably no one would build against that on Mac anymore)...  but it seems like a nice to have feature if you happen to know.
If not, we can take as is.

**Comment by janiversen** [comment]
Yes it seems macdeployqt was changed in 5.11.1, due to users complaining. But this patch needs to be changed since 5.11 is no longer supported (see #2382).

**Comment by janiversen** [comment]
Will submit a new patch, when I have finished testing both 5.12.5 and 5.13.2

**Pull request closed without merging**